index.js

```diff
@@ -0,0 +1,2 @@
+const userName = prompt('Enter your name');
+alert(`Hello ${userName}!`);
```

Comment:
in this case, do not use string interpolation
```suggestion
alert('Hello ' + userName);
```